great_expectations/checkpoint/actions.py

```diff
@@ -87,7 +87,7 @@ def _build_renderer(config: dict) -> Renderer:
         )
     return renderer
 
-
+@public_api
```

Comment:
**Note for technical reviewer**: I added this to resolve an error about too many items missing the public_api decorator

Reply:
Ah amazing I was attempting to do the same thing here: https://github.com/great-expectations/great_expectations/pull/10752

Let's take in your change and then I'll update my PR to fill in any other gaps if we have them